data-es2016plus.js

```diff
@@ -914,7 +934,16 @@ exports.tests = [
           chrome48: chrome.sharedmem,
           chrome60: true,
           chrome63: { val: false, note_id: 'chr-shared-memory-spectre' },
-          safari11: true,
+          safari11: {
+            val: false,
+            note_id: 'sf-shared-memory-spectre',
+            note_html: 'The feature was temporarily disabled to mitigate the Meltdown and Spectre CPU bugs</a>.',
```

Comment:
do we need the repeated notes? if so, can they all have the link?

also, this one in particular has a `</a>` but no `<a>`.

Reply:
You are right. It can be introduced once. Link issue is also fixed. Thank you!